projects/core/src/user/connectors/order/user-order.connector.ts

```diff
@@ -10,6 +12,9 @@ export class UserOrderConnector {
   constructor(protected adapter: UserOrderAdapter) {}
 
   public get(userId: string, orderCode: string): Observable<Order> {
+    if (this.isGuid(orderCode)) {
```

Comment:
So if I understand, the only difference for a user vs a guest is that the guest will use the guid to get the order?

Reply:
yes, it is: if anonymous user, then code is guid.